board/templates/base.html

```diff
@@ -1,50 +1,130 @@
-                      rel="stylesheet" media="screen">
-                <link href="{{ STATIC_URL }}css/main.css"
-                      rel="stylesheet" media="screen"> -->
-            {% endblock head_css_site %}
-            {% block head_css_section %}{% endblock head_css_section %}
-            {% block head_css_page %}{% endblock head_css_page %}
-        {% endblock head_css %}
-    </head>
+	{# Make modifiable head elements #}
+	<head>
+		<title>{% block title %}Home {% endblock title %} | Recboard</title>
+		{% block head_favicon %}
+			<link rel="icon" type="image/png"
+				  href="{{ STATIC_URL }}images/favicon.ico">
+		{% endblock head_favicon %}
+		{% block head_meta %}
+			{% block head_meta_charset %}
+				<meta http-equiv="Content-Type"
+					  content="text/html; charset=utf-8" />
+			{% endblock head_meta_charset %}
+			{% block head_meta_contentlanguage %}
+				<meta http-equiv="Content-Language" value="en-US" />
+			{% endblock head_meta_contentlanguage %}
+			{% block head_meta_viewport %}
+				<meta name="viewport"
+					  content="width=device-width, initial-scale=1.0">
+			{% endblock head_meta_viewport %}
+		{% endblock head_meta %}
+		{% block head_css %}
+			{% block head_css_site %}
+			   <link href="{{STATIC_URL}}/static/css/bootstrap.css" type="text/css" rel="stylesheet"/>
+			   <link href="{{STATIC_URL}}/static/css/board.css" type="text/css" rel="stylesheet"/>
+			{% endblock head_css_site %}
+			{% block head_css_section %}{% endblock head_css_section %}
+			{% block head_css_page %}{% endblock head_css_page %}
+		{% endblock head_css %}
+		{% block head_js %}
+			<script language="JavaScript"  src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.0/jquery.min.js"></script>
+			<script type="text/javascript" src="{{ STATIC_URL }}/static/js/bootstrap.js"></script>
+		{% endblock head_js %}
+	</head>
 
-    <body>
-        {# Navigation Menu #}
-        {# The Document Body #}
-        <div id="site-wrapper">
-            {% block content %}
-                This is base html, if you see this, something is wrong!
-            {% endblock content %}
-        </div>
-        {# The Footer #}
-        <div class="footer">
-            <div class="container">
-                <p>Recboard Footer</p>
-            </div>
-        </div>
-    </body>
+	<body>
+		{# Navigation Menu #}
+			<nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
+			  <div class="container">
+				<a class="navbar-brand" href="#">recboard</a>
+				<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+				  <span class="navbar-toggler-icon"></span>
+				</button>
+				<div class="collapse navbar-collapse" id="navbarResponsive">
+				  <ul class="navbar-nav ml-auto">
+					<!-- <li class="nav-item active">
+					  <a class="nav-link" href="#">Home
```

Comment:
Nit: can we remove these comments? Since we have a <li> below, we can duplicate that in future if needed